config/develop/synapse-login-scipoolprod.yaml

```diff
@@ -16,6 +16,7 @@ parameters:
   EC2KeyName: 'scipool'
   VpcName: "cesspoolvpc"
   SynapseOauthClientSecret: !ssm /synapse-login-scipoolprod/SynapseOauthClientSecret
+  PropertiesFilename: "scipooldev.properties"
```

Comment:
why is 'prod' in the filename for the develop config,  isn't that confusing?

Reply:
good catch, it should be dev